docs/manualGrading/index.md

```diff
@@ -94,10 +94,61 @@ The user interface for manual grading is built to encourage a workflow where all
+
+Instructors may specify a rubric to use positive or negative grading. In positive grading, submissions start at zero, and individual rubric items correspond to requirements to be added to the score. In negative grading, submissions start at the maximum score, and individual rubric items correspond to penalties subtracted from the score. Either option allows the creation of items with no points, which provide feedback that does not affect score. Additionally, rubric items may be set to a combination of positive and negative items, at the discretion of the grader.
+
+By default, the score produced by the rubric items is capped at the maximum points associated to the question, and penalties do not cause the points to get below zero. This limit can optionally be extended by assigning a minimum rubric score (typically to allow negative scores) and maximum extra credit to the rubric (to allow points above the maximum).
+
+Each rubric item is set a number of points and a description. This description is shown to both graders and in feedback to students. It is also possible to add an additional detailed explanation to the rubric item, which is shown to students upon request. Finally, a grader note can be set that is visible to graders, but not to students, and can be used to provide instructions on scenarios when the rubric should be used or not.
+
+Items can also be set to always be listed for students (default), or only if it is is selected. This option may be used in cases such as penalties that apply to a small subset of students, or for credit assigned to unconventional correct answers that are not to be advertised to students that did not use them.
+
+In the description, explanation, and grader note, it is possible to use mustache patterns based on the parameters, correct answers and submitted answers for a submission. This will cause the description of the item provided to graders and students to match the question variant assigned to the student, as shown in the screenshot below.
+
+![Rubric items in settings](rubric_items_setting.png)
+
+This will result in a grading panel containing a description with the value of the parameter replaced with the appropriate value for the student's variant.
+
+![Rubric items in grading panel](grading_panel_with_rubrics.png)
+
+Student's feedback is also replaced in a similar manner.
+
+![Rubric items in student feedback](student_feedback_with_rubrics.png)
+
+As grading progresses, new scenarios may be found by graders that require changes in the rubric, such as new types of penalties or credits or specific kinds of feedback. The rubric may be changed as required, even after grading is already underway. These changes may affect existing submissions:
```

Comment:
```suggestion
Rubrics in PrairieLearn are dynamic. As grading progresses, new scenarios may be found by graders that require changes in the rubric, such as new types of penalties or credits or more specialized feedback. The rubric may be changed as needed even after grading is already underway - all graders will see changes to the rubric in real time. 

Below is a list of how changes to the rubric will affect affect submissions that have already been manually graded:
```

Reply:
> "all graders will see changes to the rubric in real time"

Is this true? I can't remember if they need to refresh, or move to a new submission, or if it will alert them to a change.

Reply:
> Is this true? I can't remember if they need to refresh, or move to a new submission, or if it will alert them to a change.

Other than the grader that updated the rubric, other graders need to refresh.

```suggestion
### Updating the rubric

As grading progresses, new scenarios may be found by graders that require changes in the rubric, such as new types of penalties or credits or more specialized feedback. The rubric may be changed as required, even after grading is already underway. Changes in the rubric are presented to all graders in any submission that still requires grading.

If the rubric is changed after some submissions have already been manually graded, the changes may affect the previously graded submissions, as follows:
```